pkg/capability/capability.go

```diff
@@ -24,3 +90,14 @@ func PresentCapabilities(capabilities []*v1.Capability) CapabilityList {
 	}
 	return capabilitiesList
 }
+
+func GetAvailableCapability(capability string, resourceType string) (string, error) {
+	val, ok := availableCapabilities[capability]
+	if !ok {
+		return "", fmt.Errorf("capability not found")
+	}
```

Comment:
if the user didn't provide a valid capability, maybe we can show the list of capabilities or return a message pointing where to find this information in the docs/command line helper.